renovate.json

```diff
@@ -117,6 +117,56 @@
       "groupName": "react",
       "groupSlug": "pie-storybook-react-devDeps",
       "enabled": true
+    },
```

Comment:
I didn't add groups for wc-nuxt2 / wc-next10, as these are meant to replicate ConsumerWeb / Coreweb